pg_analytics/src/hooks/handler.rs

```diff
@@ -1,30 +1,29 @@
 use pgrx::*;
 use std::ffi::{c_char, CString};
 
-use crate::errors::{NotSupported, ParadeError};
+use crate::errors::ParadeError;
 
 static COLUMN_HANDLER: &str = "parquet";
 
-pub trait IsColumn {
+pub trait TableClassifier {
     #[allow(clippy::wrong_self_convention)]
-    unsafe fn is_column(self) -> Result<bool, ParadeError>;
+    unsafe fn table_lists(self) -> Result<(Vec<String>, Vec<String>), ParadeError>;
 }
 
-impl IsColumn for *mut pg_sys::List {
-    unsafe fn is_column(self) -> Result<bool, ParadeError> {
+impl TableClassifier for *mut pg_sys::List {
+    unsafe fn table_lists(self) -> Result<(Vec<String>, Vec<String>), ParadeError> {
         let oid = column_oid()?;
 
-        if oid == pg_sys::InvalidOid {
-            return Ok(false);
-        }
-
         #[cfg(feature = "pg12")]
         let mut current_cell = (*self).head;
         #[cfg(any(feature = "pg13", feature = "pg14", feature = "pg15", feature = "pg16"))]
         let elements = (*self).elements;
 
-        let mut using_noncol: bool = false;
-        let mut using_col: bool = false;
+        // let mut using_noncol: bool = false;
+        // let mut using_col: bool = false;
+
+        let mut row_tables = vec![];
+        let mut col_tables = vec![];
```

Comment:
If you're looking to leave all these commented lines in the file, could you leave a note explaining why?